core/pom.xml

```diff
@@ -78,6 +91,7 @@
                           io.stargate.core.*,
                           com.codahale.metrics,
                           com.codahale.metrics.*,
+                          io.micrometer.*
```

Comment:
Do we have to export `io.micrometer`? If it is not required, I believe it would be preferable to avoid exporting new packages. If it is required, could you explain why, please?